db/content/layouts/base.html

```diff
@@ -1,174 +1,172 @@
+          <span class="icon-facebook" data-grunticon-embed="true"></span>
+          <span class="visually-hidden">
+            Follow First Night Pittsburgh on Facebook
+          </span>
+        </a>
+      </li>
+      <li class="social-link-item">
+        <a href="https://twitter.com/culturaltrust">
+          <span class="icon-twitter" data-grunticon-embed="true"></span>
+          <span class="visually-hidden">
+            Follow First Night Pittsburgh on Twitter
+          </span>
+        </a>
+      </li>
+      <li class="social-link-item">
+        <a href="http://clicks.skem1.com/signup/?c=vNHhx&lid=45">
+          <span class="icon-envelope" data-grunticon-embed="true"></span>
+          <span class="visually-hidden">
+            Sign up for the First Night Pittsburgh Email Newsletter
+          </span>
+        </a>
+      </li>
+    </ul>
+
+  </div>
+</footer>
+<footer class="site-footer">
+  <div class="container">
+    <nav class="footer-navigation" role="navigation">
+      <ul class="general">
+        <a class="cultural-trust-logo" href="http://www.trustarts.org">
+          <img src="/production/assets/pct-black.png" alt="Trust logo" />
+          <h4 class="visually-hidden">Pittsburgh Cultural Trust</h4>
+        </a>
+        <li><a href="http://pressroom.trustarts.org/">Press Room</a></li>
+        <li><a href="http://trustarts.org/about/privacy-policy">Privacy Policy</a></li>
+        <li><a href="/kids/about/contact-us">Contact Us</a></li>
+        <li><a href="http://trustarts.org/about/privacy-policy">Legal</a></li>
+      </ul>
+      <ul class="attending">
+        <li class="footer-navigation-header">Attending</li>
+        <li><a href="/kids/plan-your-visit/parking-transportation/">Parking and Transportation</a></li>
+        <li><a href="/kids/plan-your-visit/festival-food/">Food</a></li>
+        <li><a href="/kids/plan-your-visit/restrooms/">Restrooms</a></li>
+        <li><a href="/kids/plan-your-visit/accessibility/">Accessibility</a></li>
+        <li><a href="/kids/plan-your-visit/faq">FAQ</a></li>
+      </ul>
+      <ul class="artists">
+        <li class="footer-navigation-header">Download</li>
+        <li><a href="/kids/plan-your-visit/maps/">Maps</a></li>
+        <li><a href="/kids/pages/schedule">Schedule</a></li>
+        <li><a href="/kids/plan-your-visit/brochure-pdf/">Brochure</a></li>
+      </ul>
+      <ul class="Blog">
+        <li class="footer-navigation-header">Blog</li>
+        <li><a href="/kids/blog/welcome-to-the-cultural-district">Welcome to the Cultural District </a></li>
+      </ul>
+    </nav>
+    <p class="copyright">
+      © 2015 EQT CHILDREN'S THEATER FESTIVAL – Pittsburgh, PA – A production of the Pittsburgh Cultural Trust.
+    </p>
+  </div>
+</footer>
+<script src="https://s3.amazonaws.com/festivity-prod/omniture/kids/AppMeasurement.js" type="text/javascript"></script>
+<r:omniture_vars />
+</div>
+</div>
+</body>
+</html>
```

Comment:
I did. Was keeping them there until I had a sense of what was going to work, but then didn't loop back because I failed to make a ticket for myself to do so. :frowning:  Traf's is currently in test, first night's is in production and test. 

Both TRAF and First Night have their logos in snippets, too, it's worth noting.